docs/visual-testing/integrations/java.md

```diff
@@ -424,17 +428,177 @@ import com.saucelabs.visual.model.FullPageScreenshotConfig;
 
 CheckOptions options = new CheckOptions();
 FullPageScreenshotConfig config = new FullPageScreenshotConfig.Builder()
-        .withDelayAfterScrollMs(500)
-        .withDisableCSSAnimation(false)
-        .withHideAfterFirstScroll("#header")
-        .withHideScrollBars(false)
-        .withScrollLimit(5)
-        .build();
+    .withDelayAfterScrollMs(500)
+    .withDisableCSSAnimation(false)
+    .withHideAfterFirstScroll("#header")
+    .withHideScrollBars(false)
+    .withScrollLimit(5)
+    .build();
 options.enableFullPageScreenshots(config);
 visual.sauceVisualCheck("Long content page", options);
 ```
 
-<FullPageLimit />
+#### Mobile Native (beta)
+
+Methods available:
+- `withDelayAfterScrollMs(int delayAfterScrollMs)`: Delay in ms after scrolling and before taking screenshots. The default value is 0. We recommend using this option for lazy loading content.
+- `withNativeClipSelector(SelectorIn nativeClipSelector)`: Selector used to identify the first element to which clipping will be applied.
+- `withScrollElement(WebElement scrollElement)`: Scrollable element used for scrolling. The default is root element.
+- `withScrollLimit(int scrollLimit)`: Limit the number of screenshots taken for scrolling and stitching. The default value is 10. The value needs to be between 1 and 10.
+
+:::note
+It is recommended to define the `withScrollElement` as the appropriate scrollable container.
+:::
+
```

Comment:
When looking to the rendered doc (https://docs.dev.saucelabs.net/pr-preview/pr-3025/visual-testing/integrations/java/#mobile-native-beta), this is not super clear what are those 3 blocks of code.

It took me several seconds to understand that this was actually different examples, with different configurations. We should definitively add text explaining what a customer is seeing and/or a title on top of it.

Otherwise that's probably not clear enough.